### PR TITLE
New Geo database. Finished and working.

### PR DESCRIPTION
**Pull request by webmods-croatia**
@navjottomer Please check this. No idea why it can't merge automatically. Is develop branch fine?

https://github.com/navjottomer/Osclass/issues/68

### CODE REVIEW

File: oc-includes/osclass/helpers/hUtils.php [hunk 1]
```diff
@@ -306,5 +306,22 @@ function osc_get_subdomain_params() {
...
+}
 
+/**
+ * Get URL of location SQL.
+ *
+ * @param string $location
+ * @return string
+ */
+function osc_get_locations_sql($location) {
+    return 'https://raw.githubusercontent.com/webmods-croatia/love-osclass/master/locations/'.$location;
```
Comment (navjottomer) [inline oc-includes/osclass/helpers/hUtils.php]: Need to change this URL to Osclass Extras repo.

**Comment by webmods-croatia** [comment]
@navjottomer But that isn't a direct link?

**Comment by webmods-croatia** [comment]
Sorry, misunderstanding. Deleted my old comment.

Will you fix or I need to make new PR?

**Comment by navjottomer** [comment]
Make a new PR that is enough.

**Comment by webmods-croatia** [comment]
Ok. Thanks. :)

**Comment by webmods-croatia** [comment]
@navjottomer https://github.com/navjottomer/Osclass/pull/77

**Pull request merged**